private/pkg/git/testdata/packed-refs

```diff
@@ -1,244 +1,8 @@
 # pack-refs with: peeled fully-peeled sorted 
-e1ce94c9dd3a187478e672d73e5d8a31fbf4a21b refs/remotes/origin/amckinney/template-plugin-DO-NOT-DELETE
-102fd69fd306f2bc6e875ffa7d58a1fb4337d9ba refs/remotes/origin/automated-changelog
-837b8eb6a1ec4799a95350a726774c2e55b73cc4 refs/remotes/origin/buf-beta-price-organization
-31dc47f8d9625b229f72fd5a983880d3aaf6c3aa refs/remotes/origin/bufgenv2
-4d3dedec75a6efcb60827c9054056d88c768b4b0 refs/remotes/origin/croche/bsr-1614-dogfood-remote-packages-in-cli
-ba5ca857e8522eb8be435cc8a926014754139c43 refs/remotes/origin/cyinma/deprecated-curated-plugin
-a3265eb607323e8ca2771b52f2fc5486ecaeadfc refs/remotes/origin/do-not-delete-this-branch-comments-exclude-request-responses
-ed8a55932ff3f77304e9acadf1670415c40c77d6 refs/remotes/origin/do-not-delete-this-branch-format-stdin
-f7c6405aff1684b25a78740e87508bd8a6a2eba2 refs/remotes/origin/do-not-delete-this-branch-plugin-extensions
-ea2e60b5a7a06de033219e695fc3a6ca33e0b45e refs/remotes/origin/ejackson/lint-poc
-2cfc2bb386bde9cae3830b0e99ae448256338b6e refs/remotes/origin/example-validate-rule
-a59b57ae4a01d6171afc29458293324d2833f748 refs/remotes/origin/fix-breaking-workspace
-55e5b99a957c38c993c18c29047bdd3b6be5292d refs/remotes/origin/fsnotify
-62bc3be78d1bf38e57aa2aae1aa0077a0b962eb5 refs/remotes/origin/jfigueroa/module-pin-string-parser
 45c2edc61040013349e094663e492996e0c044e3 refs/remotes/origin/main
 1fddd89116e24df213d43b7d837f5dd29ee9cbf0 refs/remotes/origin/paralleltest
-dfe0ff6db8bc524976b2f9b65dfe9105a665e0ce refs/remotes/origin/proto3-optional-oneof-breaking
-eabe913e1653b05ed0fee809c9c5a6f52948b3c3 refs/remotes/origin/release-improvement
-f9500f579132152e7129a499629a24ec1616e684 refs/remotes/origin/robbert/wasm-plugins-in-path
-13832b664214c7746c140d23773e6f9640b08857 refs/remotes/origin/saquib/bsr-1760-scaffold-workspace-push-command
-9bd7f631281cd67685458e49e54cc85f9a7e527e refs/remotes/origin/twilly/git-storage-reader-v2
+45c2edc61040013349e094663e492996e0c044e3 refs/remotes/otherorigin/main
+1fddd89116e24df213d43b7d837f5dd29ee9cbf0 refs/remotes/otherorigin/paralleltest
```

Comment:
Other nit, ideally the remotes in other origins should have different name or hash or both, so when reading the test we're making sure we're reading them from origin and not other remotes, or not deduplicating.